.circleci/ctest_driver_script.cmake

```diff
@@ -28,4 +31,4 @@ ctest_test(RETURN_VALUE test_status CAPTURE_CMAKE_ERROR cmake_errors)
 if (NOT "${test_status}" EQUAL 0)
   message(SEND_ERROR "some tests did not pass cleanly")
 endif()
-ctest_submit(PARTS Test Done)
```

Comment:
Was this intentional? We want to submit the `Done` part for GitHub checks.

Reply:
It is intentional for the time being. I have a patch that I have to apply to `apt install` the latest CMake; it is currently CMake 3.7.1 or something like that which is the reason for the temporary removal.